src/gfx/tests/data.hpp

```diff
@@ -1,7 +1,7 @@
-#ifndef GFX_TESTS_TEST_DATA
-#define GFX_TESTS_TEST_DATA
+#ifndef GFX_TESTS_DATA
```

Comment:
lol https://stackoverflow.com/a/32693983

Reply:
https://marketplace.visualstudio.com/items?itemName=akiramiyakoda.cppincludeguard
This extension works pretty well, I'm thinking of using the GUID's instead of the filenames too since it's even easier with renames

Reply:
guid is a bit silly/naïve though imho :)

Reply:
but yeah collisions might be an issue at some point, although you likely notice immediately so the problem is not worth a solution ;)